### PR TITLE
EQM: Section re-ordering moved to side panel, remove from section editor

### PR DESCRIPTION
**Pull request by nucleogenesis**
## Summary

Putting this draft up for feedback - will aim to fix issues around SectionEditor in this PR tomorrow.

https://github.com/learningequality/kolibri/assets/6356129/d0a6ab14-3498-48d7-9a99-53c31441dbee

## References

…

## Reviewer guidance

…

----

## Testing checklist

- [ ] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

## PR process

- [ ] PR has the correct target branch and milestone
- [ ] PR has 'needs review' or 'work-in-progress' label
- [ ] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

## Reviewer checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

**Comment by radinamatic** [comment]
Recapping what we already mentioned during the sync-up:

* Make sure we have an H1 in the sidebar
   * if not time consuming, put that H1 in the topbar
   * suggestion to concatenate the existing strings ``Edit `` + ``-`` + `` Section order``
* Align the ``Apply settings`` button right
* Make sure there is only one ``Current section``

1 | 2
-- | --
 |

**Comment by radinamatic** [comment]
* ✔️ for the H1
* No problem to keep the ``Section order`` within the panel and not in the topbar, but could we at least insert the ``Edit`` string in front, as I suggested above?
* I again managed to replicate the 3 current sections: it happens when the order panel is opened, and page reloaded at that point; when loading finishes, all 3 sections are marked as current. This persists even after the quiz is saved and closed 😳 

1 | 2
-- | --
 |

**Review by marcellamaki — APPROVED** [review]
I've tested out the problems that Radina ran into, and can't reproduce them. Code changes look okay (I did add one small non-blocking comment)

File: kolibri/plugins/coach/assets/src/composables/quizCreationSpecs.js [hunk 1]
```diff
@@ -117,7 +117,7 @@ export const QuizQuestion = {
 export const QuizSection = {
   section_id: {
     type: String,
-    default: () => String(performance.now()),
+    default: () => Math.random().toString(36).substring(7),
```
Comment (marcellamaki) [inline kolibri/plugins/coach/assets/src/composables/quizCreationSpecs.js]: non-blocking, this is not super easy to read

Comment (nucleogenesis) [reply kolibri/plugins/coach/assets/src/composables/quizCreationSpecs.js]: Thanks - I added a comment. It's a snippet I've used before to get a random-enough string. Do you think that helps?

**Pull request merged**